### PR TITLE
Community patches roll-up

### PR DESCRIPTION
**Pull request by skull-squadron**
Boils down community improvements into a tiny mega PR.

### CODE REVIEW

**Comment by cardoe** [comment]
You will need to bump the minimum Rust version in `.travis.yml` to match what `num_traits` requires, which based on the errors appears to be 1.8.0. I think at this point its ok to do so given how old 1.0.0 and 1.8.0 are.

**Comment by skull-squadron** [comment]
Could you edit that? I'm far away from any laptop.

File: src/lib.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@
 //! ```
 
 
-extern crate num_traits;
+pub extern crate num_traits;
```
Comment (cardoe) [inline src/lib.rs]: Is this really necessary to be public? We only need `FromPrimitive` which is already public.

**Comment by cardoe** [comment]
I'm not the author nor do I have commit access.

**Comment by skull-squadron** [comment]
@cardoe Oh well, thanks for noticing it. I was in the desert, but not doing Burning Man this year though. I'll play this for myself cryin' in my beer ;) 
insert tiny violin

🤘

**Review by cardoe — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (cardoe) [reply src/lib.rs]: Noticed the same behavior is done in #11 but I still don't think its needed.

**Comment by skull-squadron** [comment]
Yes, it doesn't appear to add anything of value.

File: README.md [hunk 2]
```diff
@@ -28,8 +28,7 @@ wrap your `enum` declaration inside the `enum_from_primitive!` macro.
 
 ```rust
 #[macro_use] extern crate enum_primitive;
-extern crate num;
-use num::FromPrimitive;
+use enum_primitive::num_traits::FromPrimitive;
```
Comment (cardoe) [inline README.md:31]: `use enum_primitive::FromPrimitive;`

File: src/lib.rs [hunk 3]
```diff
@@ -31,9 +31,9 @@
 //! # Example
 //!
 //! ```
-//! #[macro_use] extern crate enum_primitive;
-//! extern crate num_traits;
-//! use num_traits::FromPrimitive;
+//! #[macro_use]
+//! extern crate enum_primitive;
+//! use enum_primitive::num_traits::FromPrimitive;
```
Comment (cardoe) [inline src/lib.rs:36]: `use enum_primitive::FromPrimitive;`

**Pull request closed without merging**